### ISSUE TITLE
Exit transition occurs too early when used with suspense.

### ISSUE THREAD

**Issue by minht11** [otonashixav/solid-flip#6]
When using `TransitionGroup` and [`solid-app-router`](https://github.com/solidjs/solid-app-router) with suspense and lazy components exit transition occurs too early.

This is regression, the last working version was 0.7.5.

[Reproduction](https://codesandbox.io/s/youthful-bose-3uo9e?file=/index.js)
See when clicking link that transition occurs instantly despite route still being lazily loaded. Transition should not occur until new route is ready.

Sample code:
```tsx
const HomeLazy = lazy(async () => {
  await wait(1000)

  return { default: () => <div>Home</div> }
})

const ROUTES = [
  {
    path: "/",
    component: () => <div>Base</div>
  },
  {
    path: "/home",
    component: HomeLazy
  }
];
...
<Suspense>
    <TransitionGroup
      enter={animateEnter({
        keyframes: {
          opacity: [0, 1],
        },
        options: { duration: 2000, easing: 'linear' },
      })}
      exit={animateExit({
        keyframes: {
          opacity: [1, 0],
        },
        options: { duration: 2000, easing: 'linear' },
      })}
    >
      <Routes />
    </TransitionGroup>
  </Suspense>
```

**Comment by otonashixav** [comment]
I'd have to dig into how suspense works in order to fix this; seems that `createComputed` ignores(?) suspense. Here's a workaround that hopefully doesn't introduce any issues: https://codesandbox.io/s/tender-bell-kxtuv?file=/index.js

**Comment by otonashixav** [comment]
Fixed in 0.10.3
